### ISSUE TITLE
Underscores are generated for multiline statements when not necessary

### ISSUE THREAD

**Issue by jnm2** [paul1956/CSharpToVB#13]
Source line: https://github.com/dotnet/roslyn/blob/c0a1d1ec94efc2b4fcc2334580c84741e10e45ef/src/EditorFeatures/CSharpTest/SimplifyInterpolation/SimplifyInterpolationTests.cs#L23

Expected:

```vb
        <Fact>
```

Actual:
```vb
        <Fact> _
```

-----

Source lines: https://github.com/dotnet/roslyn/blob/c0a1d1ec94efc2b4fcc2334580c84741e10e45ef/src/EditorFeatures/CSharpTest/SimplifyInterpolation/SimplifyInterpolationTests.cs#L38-L39

Expected:

```vb
            Assert.Equal(
                {
```

Actual:

```vb
            Assert.Equal( _
                { _
```

**Comment by paul1956** [comment]
There is/was a note in the readme about this, VS removed them then you open it. I put them there when there is a possibility of a blank line that C# allows but VB does not. I am open to looking at this again.

**Comment by paul1956** [comment]
Many are fixed in 4.2.0.0, open new issue if you see more
